pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -12,12 +12,209 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import os
+from abc import ABC
 from pytorch_lightning.callbacks import ModelCheckpoint, EarlyStopping, ProgressBarBase, ProgressBar
 from pytorch_lightning.utilities import rank_zero_warn
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 
 
-class CallbackConnector:
+class CallbackConnectorLoggingMixin(ABC):
```

Comment:
So are really using mixins again and even in combination with Connectors?

Reply:
Hey @Borda @williamFalcon. Do we want to remove the use of ` Mixin` entirely, or they can be used to re-group connectors logic ? IMO, I like them as they are great for readability.

Reply:
I also think this is more readible. But on the other hand it may be confusing when to use mixins and when to avoid them.